index.html

```diff
@@ -22,17 +22,37 @@
     <div class="frame">
       <div class="slideshow-container">
         <div class="mySlides fade">
-          <img src="./images/maps/BigWorld.png">
+          <img src="./images/TripleA.png">
+          <div class="text">TripleA</div>
+        </div>
+        <div class="mySlides fade">
+          <img src="./images/maps/high-res/BigWorld.png">
           <div class="text">Big World</div>
         </div>
         <div class="mySlides fade">
-          <img src="./images/maps/270BC.png">
+          <img src="./images/maps/high-res/270BC.png">
           <div class="text">270BC</div>
         </div>
         <div class="mySlides fade">
-          <img src="./images/maps/GreatWar.png">
+          <img src="./images/maps/high-res/ClassicWW2-3rd.png">
```

Comment:
Avoid committing binary files to github. Use the github file hosting service to do this instead. Open a new issue, upload an image, and use the link it creates. You do not even need to submit the image.